### PR TITLE
Razer Cheryl Overlays

### PR DESCRIPTION
**Pull request by linuxct**

### CODE REVIEW

**Comment by linuxct** [comment]
Opened new pull. Messed up the last one because of a commit I accidentally picked when rebasing.

File: Razer/cheryl/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.razer.cheryl"
+        android:versionCode="1"
+        android:versionName="1.0">
+	<!--
+		TESTS: Ignore ro.vendor.product.name
```
Comment (phhusson) [inline Razer/cheryl/AndroidManifest.xml]: Moto G6 has a link to stock ROM explaining why.

In this case, I really don't see why you're doing this, here is the link to cheryl's vendor build.prop
https://github.com/TadiT7/razer_cheryl_dump/blob/cheryl-user-8.1.0-OPM1.171019.011-RZR-180509-5038-release-keys/vendor/build.prop
You should be able to just match on ro.vendor.build.fingerprint

File: Razer/cheryl/res/values/arrays.xml [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        <item>4000</item>
+        <item>10000</item>
+    </integer-array>
+    <integer-array name="config_defaultNotificationVibePattern">
+        <item>0</item>
+        <item>45</item>
+        <item>250</item>
+        <item>45</item>
+    </integer-array>
+    <string-array name="config_disabledUntilUsedPreinstalledImes">
```
Comment (phhusson) [inline Razer/cheryl/res/values/arrays.xml]: That's default AOSP value, and that is definitely not linked to hardware

File: Razer/cheryl/res/values/arrays.xml [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+        <item>com.android.inputmethod.latin</item>
+    </string-array>
+    <string-array name="config_ephemeralResolverPackage">
+        <item>com.google.android.gms</item>
```
Comment (phhusson) [inline Razer/cheryl/res/values/arrays.xml]: Well that's gapps specific, and not hardware dependant.

File: Razer/cheryl/res/values/arrays.xml [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+    </string-array>
+    <string-array name="config_globalActionsList">
```
Comment (phhusson) [inline Razer/cheryl/res/values/arrays.xml]: That's ROMs choice, not device.

File: Razer/cheryl/res/values/arrays.xml [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+        <item>umts:4094,87380,1220608,4096,16384,1220608</item>
+        <item>hspa:4094,87380,1220608,4096,16384,1220608</item>
+        <item>hsupa:4094,87380,1220608,4096,16384,1220608</item>
+        <item>hsdpa:4094,87380,1220608,4096,16384,1220608</item>
+        <item>hspap:4094,87380,1220608,4096,16384,1220608</item>
+        <item>edge:4093,26280,35040,4096,16384,35040</item>
+        <item>gprs:4092,8760,11680,4096,8760,11680</item>
+        <item>evdo:4094,87380,524288,4096,16384,262144</item>
+    </string-array>
+    <string-array name="config_nonBlockableNotificationPackages">
```
Comment (phhusson) [inline Razer/cheryl/res/values/arrays.xml]: That's not device-specific

File: Razer/cheryl/res/values/arrays.xml [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+        <item>com.android.deskclock</item>
+    </string-array>
+    <integer-array name="config_notificationFallbackVibePattern">
+        <item>0</item>
+        <item>35</item>
+        <item>150</item>
+        <item>35</item>
+    </integer-array>
+    <string-array name="config_notificationSignalExtractors">
+        <item>com.android.server.notification.NotificationChannelExtractor</item>
```
Comment (phhusson) [inline Razer/cheryl/res/values/arrays.xml]: that's not hardware specific, and that's AOSP default

File: Razer/cheryl/res/values/arrays.xml [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+    <string-array name="config_tether_wifi_regexs">
+        <item>wigig0</item>
+        <item>softap0</item>
+    </string-array>
+    <integer-array name="config_virtualKeyVibePattern">
+        <item>0</item>
+        <item>13</item>
+        <item>10</item>
+    </integer-array>
+    <string-array name="config_vvmSmsFilterRegexes"> <item>310004,310010,310012,310013,310590,310890,310910,311110,311270,311271,311272,311273,311274,311275,311276,311277,311278,311279,311280,311281,311282,311283,311284,311285,311286,311287,311288,311289,311390,311480,311481,311482,311483,311484,311485,311486,311487,311488,311489;^//VZWVVM.*</item>
```
Comment (phhusson) [inline Razer/cheryl/res/values/arrays.xml]: that's AOSP default

File: Razer/cheryl/res/values/integers.xml [hunk 8]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<integer name="config_previousVibrationsDumpLimit">20</integer>
```
Comment (phhusson) [inline Razer/cheryl/res/values/integers.xml]: i'm not sure that's hardware specific, but whatever

**Comment by phhusson** [comment]
Merged in https://github.com/phhusson/vendor_hardware_overlay/commit/2c5709b430f2fbd03591dd3479de266d694a4613

**Pull request closed without merging**